Apps/Admin/Client/Components/Details/ProfileTab.razor

```diff
@@ -0,0 +1,44 @@
+﻿@using HealthGateway.Common.Data.Utils
+@using HealthGateway.Admin.Client.Components.Support
+@inherits Fluxor.Blazor.Web.Components.FluxorComponent
+
+@if (Patient != null)
+{
+    <MudText Class="mb-4 mt-6" Typo="Typo.h6">
+        Patient Details
+    </MudText>
+    <MudGrid Spacing="2">
+        <MudItem xs="12" lg="4">
+            <HgField Label="Name" Value="@PatientName" data-testid="patient-name" />
+        </MudItem>
+        <MudItem xs="12" lg="4">
+            <HgField Label="Birthdate" Value="@Patient.Birthdate.ToString()" data-testid="patient-dob" />
+        </MudItem>
+        <MudItem xs="12" lg="4">
+            <HgField Label="PHN" Value="@Patient.PersonalHealthNumber" data-testid="patient-phn" />
+        </MudItem>
+        <MudItem xs="12" lg="6">
+            <HgField Label="Physical Address" Value="@AddressUtility.GetAddressAsSingleLine(Patient.PhysicalAddress)" data-testid="patient-physical-address" />
+        </MudItem>
+        <MudItem xs="12" lg="6">
+            <HgField Label="Mailing Address" Value="@AddressUtility.GetAddressAsSingleLine(Patient.PostalAddress)" data-testid="patient-mailing-address" />
+        </MudItem>
+    </MudGrid>
+
+    if (CanViewCovidDetails && !ImmunizationsAreBlocked)
+    {
+        <MudText Class="mb-4 mt-6" Typo="Typo.h6">
+            Covid-19 Section
+        </MudText>
```

Comment:
The acceptance criteria mentions this should be collapsible.  I'd also suggest changing the label to "COVID-19" (dropping "Section" and using appropriate capitalization).

Reply:
Thanks good catch